src/components/PostEditor/PostEditor.js

```diff
@@ -429,6 +429,9 @@ export default class PostEditor extends React.Component {
             <MemberSelector
               initialMembers={members || []}
               onChange={this.updateProjectMembers}
+              // NOTE: Turn-on to limit Project Member selection
```

Comment:
i say lets turn this on. i dont actually understand how event/project invites currently work with people outside the communities it is posted to. Can they see the post even though they are not in any of the communities it is posted to?

Reply:
Yes, I think this was always a known gap and due to the lift required--which I did this week--we had let it slip and were planning to loop back. Now that this is done it makes sense to catch it up here. * However, the tidy part of my brain wants to push this branch/PR and then apply this to projects as a separate branch/PR (I can get that out pretty quickly on Monday).